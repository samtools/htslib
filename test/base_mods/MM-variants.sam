@SQ	SN:chr1	LN:2000
fwd-implicit-has-list	0	chr1	1010	20	7M	*	0	0	ATGCATC	abcdefg	MM:Z:C+m.,0,0;G-a.,0;
rev-implicit-has-list	16	chr1	1020	20	7M	*	0	0	ATGCATG	abcdefg	MM:Z:C+m.,0,0;G-a.,0;
fwd-explicit-has-list	0	chr1	1030	20	7M	*	0	0	ATGCATC	abcdefg	MM:Z:C+m?,0,0;G-a?,0;
rev-explicit-has-list	16	chr1	1040	20	7M	*	0	0	ATGCATG	abcdefg	MM:Z:C+m?,0,0;G-a?,0;
fwd-implicit-empty	0	chr1	1050	20	7M	*	0	0	ATGCATC	abcdefg	MM:Z:C+m.;G-a.;
rev-implicit-empty	16	chr1	1060	20	7M	*	0	0	ATGCATG	abcdefg	MM:Z:C+m.;G-a.;
fwd-explicit-empty	0	chr1	1070	20	7M	*	0	0	ATGCATC	abcdefg	MM:Z:C+m?;G-a?;
rev-explicit-empty	16	chr1	1080	20	7M	*	0	0	ATGCATG	abcdefg	MM:Z:C+m?;G-a?;
