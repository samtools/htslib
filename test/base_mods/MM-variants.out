0	A
1	T
2	G	G-a.
3	C	C+m.
4	A
5	T
6	C	C+m.
---
Present: m. a.
2	G	G-a.
3	C	C+m.
6	C	C+m.

===

0	A
1	T
2	G	C+m.
3	C	G-a.
4	A
5	T
6	G	C+m.
---
Present: m. a.
2	G	C+m.
3	C	G-a.
6	G	C+m.

===

0	A
1	T
2	G	G-a.
3	C	C+m.
4	A
5	T
6	C	C+m.
---
Present: m? a?
2	G	G-a.
3	C	C+m.
6	C	C+m.

===

0	A
1	T
2	G	C+m.
3	C	G-a.
4	A
5	T
6	G	C+m.
---
Present: m? a?
2	G	C+m.
3	C	G-a.
6	G	C+m.

===

0	A
1	T
2	G
3	C
4	A
5	T
6	C
---
Present: m. a.

===

0	A
1	T
2	G
3	C
4	A
5	T
6	G
---
Present: m. a.

===

0	A
1	T
2	G	G-a#
3	C	C+m#
4	A
5	T
6	C	C+m#
---
Present: m? a?
2	G	G-a#
3	C	C+m#
6	C	C+m#

===

0	A
1	T
2	G	C+m#
3	C	G-a#
4	A
5	T
6	G	C+m#
---
Present: m? a?
2	G	C+m#
3	C	G-a#
6	G	C+m#

===

